result/bmpserver.yaml

```diff
@@ -0,0 +1,129 @@
+components:
+  schemas:
+    BmpServer.State.Request:
+      description: >-
+        The request for BMP server information.
+      type: object
+      properties:
+        bmp_server_names:
+          description: >-
+            The names of BMP server to return results for.
+            An empty list will return results for all BMP server information.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Bmp.Server/properties/name"
+          x-field-uid: 1
+
+    BmpServer.State:
+      description: >-
+       The information associated with BMP server.
+      type: object
+      properties:
+        bmp_server_name:
+          description: >-
+            The name of a BMP server.
+          type: string
+          x-field-uid: 1
+        information_string:
+          description: >-
+            The administratively assigned UTF-8 string information carried in Information field of 
+            Information TLV in Initiation message.
+          type: array
+          items:
+            type: string
+          x-field-uid: 2
+        system_name:
+          description: >-
+            The Information sysName MIB-II [RFC1213] object information carried in Information field of 
+            Information TLV in Initiation message.
+          type: string
+          x-field-uid: 3
+        system_description:
+          description: >-
+            The Information sysDesc MIB-II [RFC1213] object information carried in Information field of 
+            Information TLV in Initiation message.
+          type: string
+          x-field-uid: 4
+        termination_string:
```

Comment:
Should we have these within a state = up/down and down might have these reasons while up has info per peers ? 
Or maybe return information for up peers at that time , like BGP ?